Formula/rgbds.rb

```diff
@@ -20,19 +20,32 @@ class Rgbds < Formula
 
   depends_on "bison" => :build
   depends_on "pkg-config" => :build
+  depends_on "rust" => :build
   depends_on "libpng"
 
+  resource "rgbobj" do
+    url "https://github.com/gbdev/rgbobj/archive/refs/tags/v0.1.0.tar.gz"
+    sha256 "359a3504dc5a5f7812dfee602a23aec80163d1d9ec13f713645b5495aeef2a9b"
+  end
+
   def install
     system "make", "install", "PREFIX=#{prefix}", "mandir=#{man}"
+    resource("rgbobj").stage do
+      system "cargo", "install", *std_cargo_args
+      man1.install "rgbobj.1"
+    end
+    zsh_completion.install "contrib/zsh_compl"
```

Comment:
I think this will install the directory `zsh_compl` into the `zsh_completion` directory, which I don't think is what we want.

Reply:
Ah, I thought it would install all files in that directory. I'm not very familiar with Ruby, what would be the best way to do this with each file in the directory?

Reply:
```suggestion
    zsh_completion.install Dir["contrib/zsh_compl/_*"]
```
Should install all files that match the glob in `Dir`.